recipes/libevent/meta.yaml

```diff
@@ -0,0 +1,77 @@
+{% set version = "2.0.22" %}
+
+package:
+  name: libevent
+  version: {{ version }}
+
+source:
+  fn: libevent-{{ version }}.tar.gz
+  url: https://github.com/libevent/libevent/archive/release-{{ version }}-stable.tar.gz
+  sha256: ab89639b0819befb1d8b293d52047c6955f8d1c9150c2b22a0e6247930eb9128
+
+build:
+  number: 0
+  skip: True  # [win or py3k]
```

Comment:
If this is being skipped on py3k, is py3k tmux going to be pulling `libevent` from the defaults channel?